README.md

```diff
@@ -2,6 +2,28 @@
 
 Buzz is a lightweight PHP 5.3 library for issuing HTTP requests.
 
+## Instalation
+
+Package available on [Composer](https://packagist.org/packages/kriswallsmith/buzz).
+
+If you're using Composer to manage dependencies, you can use
+
+    $ composer require "kriswallsmith/buzz": "dev-master"
```

Comment:
why not just `composer require "kriswallsmith/buzz"`? Composer resolves the best version automatically.